packages/headless/src/features/commerce/query-suggest/query-suggest-actions-loader.ts

```diff
@@ -0,0 +1,58 @@
+import {AsyncThunkAction} from '@reduxjs/toolkit';
+import {AsyncThunkCommerceOptions} from '../../../api/commerce/commerce-api-client';
+import {CommerceEngine} from '../../../app/commerce-engine/commerce-engine';
+import {
+  FetchQuerySuggestionsThunkReturn,
+  StateNeededByQuerySuggest,
+} from '../../../features/commerce/query-suggest/query-suggest-actions';
+import {querySetReducer as querySet} from '../../query-set/query-set-slice';
+import {
+  clearQuerySuggest,
+  registerQuerySuggest,
+  selectQuerySuggestion,
+} from '../../query-suggest/query-suggest-actions';
+import {
+  FetchQuerySuggestionsActionCreatorPayload,
+  QuerySuggestActionCreators,
+} from '../../query-suggest/query-suggest-actions-loader';
+import {querySuggestReducer as querySuggest} from '../../query-suggest/query-suggest-slice';
+import {fetchQuerySuggestions} from './query-suggest-actions';
+
+/**
+ * The query suggest action creators for commerce.
+ */
+export interface CommerceQuerySuggestActionCreators
```

Comment:
I noticed this is prefixed with "Commerce" but `RecentQueriesActionCreators` isn't